src/main/java/coydir/ui/PersonInfo.java

```diff
@@ -54,6 +58,8 @@ public void update(Person person) {
         phone.setText("Phone number:  " + person.getPhone().value);
         address.setText("Address:  " + person.getAddress().value);
         email.setText("Email address:  " + person.getEmail().value);
+        totalLeave.setText("Total Leave: " + person.getTotalNumberOfLeaves());
+        leaveLeft.setText("Leaves Left: " + person.getTotalNumberOfLeaves());
```

Comment:
This should be `person.getLeavesLeft()`?

Reply:
Yes boss